test/common_utils.py

```diff
@@ -539,109 +402,7 @@ def make_image_pil(*args, **kwargs):
-
-    def fn(shape, dtype, device, memory_format):
-        *batch_dims, _, height, width = shape
-        return make_image(
-            (height, width),
-            color_space=color_space,
-            batch_dims=batch_dims,
-            dtype=dtype,
-            device=device,
-            memory_format=memory_format,
-        )
-
-    return ImageLoader(fn, shape=(*extra_dims, num_channels, *size), dtype=dtype, memory_format=memory_format)
-
-
-def make_image_loaders(
-    *,
-    sizes=DEFAULT_SPATIAL_SIZES,
-    color_spaces=(
-        "GRAY",
-        "GRAY_ALPHA",
-        "RGB",
-        "RGBA",
-    ),
-    extra_dims=DEFAULT_EXTRA_DIMS,
-    dtypes=(torch.float32, torch.float64, torch.uint8),
-    constant_alpha=True,
-):
-    for params in combinations_grid(size=sizes, color_space=color_spaces, extra_dims=extra_dims, dtype=dtypes):
-        yield make_image_loader(**params, constant_alpha=constant_alpha)
-
-
-make_images = from_loaders(make_image_loaders)
-
-
-def make_image_loader_for_interpolation(
-    size=(233, 147), *, color_space="RGB", dtype=torch.uint8, memory_format=torch.contiguous_format
-):
-    size = _parse_size(size)
-    num_channels = get_num_channels(color_space)
-
-    def fn(shape, dtype, device, memory_format):
-        height, width = shape[-2:]
-
-        image_pil = (
-            PIL.Image.open(pathlib.Path(__file__).parent / "assets" / "encode_jpeg" / "grace_hopper_517x606.jpg")
-            .resize((width, height))
-            .convert(
-                {
-                    "GRAY": "L",
-                    "GRAY_ALPHA": "LA",
-                    "RGB": "RGB",
-                    "RGBA": "RGBA",
-                }[color_space]
-            )
-        )
-
-        image_tensor = to_image(image_pil)
-        if memory_format == torch.contiguous_format:
-            image_tensor = image_tensor.to(device=device, memory_format=memory_format, copy=True)
-        else:
-            image_tensor = image_tensor.to(device=device)
-        image_tensor = to_dtype_image(image_tensor, dtype=dtype, scale=True)
-
-        return datapoints.Image(image_tensor)
-
-    return ImageLoader(fn, shape=(num_channels, *size), dtype=dtype, memory_format=memory_format)
-
-
-def make_image_loaders_for_interpolation(
-    sizes=((233, 147),),
-    color_spaces=("RGB",),
-    dtypes=(torch.uint8,),
-    memory_formats=(torch.contiguous_format, torch.channels_last),
-):
-    for params in combinations_grid(size=sizes, color_space=color_spaces, dtype=dtypes, memory_format=memory_formats):
-        yield make_image_loader_for_interpolation(**params)
-
-
-@dataclasses.dataclass
-class BoundingBoxesLoader(TensorLoader):
-    format: datapoints.BoundingBoxFormat
-    spatial_size: Tuple[int, int]
-    canvas_size: Tuple[int, int] = dataclasses.field(init=False)
-
-    def __post_init__(self):
-        self.canvas_size = self.spatial_size
-
-
-def make_bounding_box(
+def make_bounding_boxes(
```

Comment:
I'd be nice if we could get rid of the unused parameters / functionality. For example we shouldn't need `batch_dims` anymore? Maybe there are others. But we can do that in a follow-up PR.